config/rbac/storageconsumer_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit storageconsumers.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Will the storageconsumer CRs be created in multiple namespaces?

Reply:
No it is namespaced. This is an auto-generated rbac role created when declaring a new API.